integration_tests/pages/find/programmes.ts

```diff
@@ -8,7 +8,10 @@ export default class ProgrammesPage extends Page {
 
   shouldHaveProgrammes(programmes: Array<AccreditedProgramme>) {
     cy.get('li').each((programmeElement, programmeElementIndex) => {
-      cy.wrap(programmeElement).should('have.text', programmes[programmeElementIndex].name)
+      cy.wrap(programmeElement)
```

Comment:
Do we think it's likely that we'll need to do this all over the place if the preferred style is to the content of an element on its own line? Is it worth us extracting this into its own Cypress command if so?